CMakeLists.txt

```diff
@@ -22,16 +22,15 @@ mark_as_advanced(FORCE FORCE_ALL_SOLVER_TEST)
 option(SKIP_PRIVATE_SOLVER_TEST "Skip private solver test" OFF)
 mark_as_advanced(FORCE SKIP_PRIVATE_SOLVER_TEST)
 
-# Option to switch between Catkin and pure CMake build system
-option(QP_SOLVER_COLLECTION_STANDALONE "If ON, build without requiring Catkin and roscpp" OFF)
-
-if(NOT QP_SOLVER_COLLECTION_STANDALONE)
+if(DEFINED CATKIN_DEVEL_PREFIX)
+  set(QP_SOLVER_COLLECTION_STANDALONE OFF)
```

Comment:
I don't think we need the variable anymore if you have replaced every occurrence of it in the CMake scripts

Reply:
Updated in https://github.com/isri-aist/QpSolverCollection/pull/12/commits/f8f521d876ba4b6941cc247b1bcc31c695cc233e